packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -120,23 +117,26 @@ class AppBar extends StatelessWidget {
   /// The following elevations have defined shadows: 1, 2, 3, 4, 6, 8, 9, 12, 16, 24
   final int elevation;
 
-  /// The color to use for the app bar's material. This generally should be set
-  /// in tandem with [brightness].
+  /// The color to use for the app bar's material. Typically this should be set
```

Comment:
I understand "typically ... is" and "generally ... should", but "typically ... should" is confusing to me.